libraries/hatch-server-user-management/src/AWSCognitoClient.ts

```diff
@@ -224,4 +243,40 @@ export default class AWSCognitoClient implements UserServiceClient {
     return pemCerts;
   }
   
+  private convertAWSErrorToUserManagementError(awsError: AWSError) {
```

Comment:
Doesn't look like this uses `this`, so it can/should be a free function.

Reply:
updated